java/tests/com/google/privacy/differentialprivacy/BoundedSumTest.java

```diff
@@ -679,4 +680,299 @@ private static void testForBias(
     // equal to the raw count.
     assertThat(stats.mean()).isWithin(sampleTolerance).of(rawEntry);
   }
+
+  @Test
+  public void computeConfidenceInterval_negativeSumBounds_noClamping() {
+    sum =
+            BoundedSum.builder()
+                    .epsilon(EPSILON)
+                    .delta(DELTA)
+                    .noise(noise)
+                    .maxPartitionsContributed(12)
```

Comment:
why 12? do we really need this in this test? From the first glance default value (1) should be ok here.

Reply:
I would add method

BoundedSum.Params.Builder getBoundedSumBuilderWithFields() {
    return BoundedSum.builder()
                    .epsilon(EPSILON)
                    .delta(DELTA)
                    .noise(noise)
                    .maxPartitionsContributed(1)
                    .maxContributionsPerPartition(1)
                    .lower(-Double.MAX_VALUE)
                   .upper(Double.MAX_VALUE);
}


and I would use it everywhere where I just need to change few parameters (which differ from default)


in the setup method

sum = getBoundedSumBuilderWithFields().build();


in this test

sum = getBoundedSumBuilderWithFields().lower(-8).upper(-2).build();

so we clearly see that in order to check the need behavior in this test we need those 2 params (and others don't matter)
}